src/core/graphics/Graphics.js

```diff
@@ -1112,15 +1112,27 @@ export default class Graphics extends Container
                 const data = this.graphicsData[i];
                 const type = data.type;
                 const lineWidth = data.lineWidth;
+                const lineAlignment = data.lineAlignment;
+
+                let lineOffset = 0;
+
+                if (lineAlignment === 0.5)
+                {
+                    lineOffset = lineWidth / 2;
+                }
+                else if (lineAlignment === 1)
+                {
+                    lineOffset = lineWidth;
```

Comment:
lineAlignment should be treated as a float from 0-1. So probably lineOffset = lineWidth * lineAlignment instead.